src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -772,13 +772,10 @@
           "rules": [
             {
               "id": "BA2001",
+              "name": "LoadImageAboveFourGigabyteAddress",
```

Comment:
>"name": "LoadImageAboveFourGigabyteAddress", [](http://example.com/codeflow?start=14&length=44)

all json node ordering change are not caused by this change, we did not run the baseline for sometime, 
running re-baseline from current main will also get this change.

Reply:
thanks for the feedback, working on
1. update SDK to have those re-ordering reverted.
2. review
3. change only one ordering so that name is right after id.

Reply:
I have created a fix in SDK
[https://github.com/microsoft/sarif-sdk/pull/2420](https://github.com/microsoft/sarif-sdk/pull/2420) (please review together)
and then point binskim to that branch and run re-baseline to verify the fix.
Now this PR is clearer to review.

After all looks good to you, we should get the desired ordering from you and I will update both PR with that order.(you mentioned one name field)